File: week-6-python/lecture-6/scores.py
def main():
    scores = []
    for i in range(3):
        score = get_int()
        # Method 1 using append
        scores.append(score)
        # Method 2 using temp list
        # scores += [score]

    average = sum(scores) / len(scores)
    print(f"Average: {average:.2f}")


def get_int():
    while True:
        try:
            x = int(input("Score: "))
            if x > 0:
                return x
        except ValueError:
            print("Not an integer")


main()

File: week-2-arrays/lecture-2/scores.c
#include <cs50.h>
#include <stdio.h>

#define N 3

float average(int array[], int length);

int main(void)
{
    int scores[N];
    for (int i = 0; i < N; i++)
    {
        scores[i] = get_int("Score: ");
    }

    printf("Average: %f\n", average(scores, N));
}

float average(int array[], int length)
{
    int total = 0;
    for (int i = 0; i < length; i++)
    {
        total += array[i];
    }
    return total / (float) length;
}
